.github/workflows/pre_merge.yml

```diff
@@ -34,15 +34,11 @@ jobs:
       - name: Code quality checks
         run: tox -vv -e pre-commit-all-py310-pt1
   Unit-Test:
-    runs-on: [self-hosted, linux, x64, dev]
     needs: Code-Quality-Checks
-    timeout-minutes: 120
     strategy:
       fail-fast: false
       matrix:
         include:
-          - python-version: "3.8"
```

Comment:
Don't we need to run unit test w/ python 3.8?

Reply:
unit tests will be ran with python 3.9 and 3.10 and the unit tests with coverage will be ran for python 3.8.